hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/AbstractS3ATestBase.java

```diff
@@ -159,4 +168,63 @@ protected void assertStatusCode(AWSServiceIOException e, int code)
       throw e;
     }
   }
+
+  /**
+   * Assert that a path is encrypted with right encryption settings.
+   * @param path file path.
+   * @param algorithm encryption algorithm.
+   * @param kmsKeyArn
```

Comment:
add a javadoc entry for the argument; just delete the @throws IOE line altogether